pyrobosim/pyrobosim/navigation/astar_grid.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python3
```

Comment:
Missing docstring at beginning of file, so docs look like this: 

See the PRM/RRT for how to fix.

![image](https://user-images.githubusercontent.com/4603398/223610149-d7c45a57-26c2-4a27-97ba-6935e8a73965.png)